### ISSUE TITLE
Power monitoring

### ISSUE THREAD

**Issue by colinsauze** [abersailbot/dewi-hardware#16]
Monitor battery voltage and currents on various devices.

**Comment by colinsauze** [comment]
We are currently looking at using the INA219 at work. Its an I2C current/voltage monitor able to monitor up to 3.2A and 26V. See https://www.adafruit.com/product/904

**Comment by naturesyouth** [comment]
that looks pretty cool, definitely worth looking into.

**Comment by naturesyouth** [comment]
could this be spun into a pcb in the battery box? that way we can monitor both sides of the power system?

**Comment by Danyc0** [comment]
IIRC, based on the measurements I took last year, with both wifi dongles running, especially if there's now a Pi3 (plz no), and we're doing a lot of CV we will overrun that 3.2A limit

**Comment by naturesyouth** [comment]
I would think we would have one chip per channel / battery rather then one for everything. and was that including servos? because the power supply for the logic isn't rated for anything more then an amp and a half.

**Comment by Danyc0** [comment]
Now I come to think of it, I'm not sure :P Please disregard that comment :L

**Comment by colinsauze** [comment]
It is possible (but hard, needs surface mount soldering) to change the shunt resistor and adjust its measurement range. But 3.2A max for the whole boat sounds about right to me. If there are multiple sensors then you shouldn't saturate them.

**Comment by TomGC96** [comment]
Yeah you do not remember correctly, the measurements for current I took for the Pi, Arduino and Captain Morgan hardly reached 0.5A.
